pkgs/development/python-modules/langchain-mongodb/default.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  azure-identity,
+  buildPythonPackage,
+  fetchFromGitHub,
+  freezegun,
+  langchain,
+  langchain-core,
+  pymongo,
+  lark,
+  pandas,
+  poetry-core,
+  pytest-asyncio,
+  pytest-mock,
+  pytest-socket,
+  pytestCheckHook,
+  pythonOlder,
+  requests-mock,
+  responses,
+  syrupy,
+  toml,
+}:
+
+buildPythonPackage rec {
+  pname = "langchain-mongodb";
+  version = "0.1.6";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langchain";
+    rev = "refs/tags/langchain-mongodb==${version}";
+    hash = "sha256-p/cdWFPc2Oi5aRmjj1oAixM6aDKw0TbyzMdP4h2acG4=";
+  };
+
+  sourceRoot = "${src.name}/libs/partners/mongodb";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [
+    langchain-core
+    pymongo
+  ];
+
+  nativeCheckInputs = [
+    freezegun
+    langchain
+    lark
+    pandas
+    pytest-asyncio
+    pytest-mock
+    pytest-socket
+    pytestCheckHook
+    requests-mock
+    responses
+    syrupy
+    toml
+  ];
+
+  pytestFlagsArray = [ "tests/unit_tests" ];
+
+  pythonImportsCheck = [ "langchain_mongodb" ];
+
+  passthru = {
+    updateScript = langchain-core.updateScript;
+  };
+
+  meta = {
+    description = "An integration package connecting OpenAI and LangChain";
+    homepage = "https://github.com/langchain-ai/langchain";
+    changelog = "https://github.com/langchain-ai/langchain/releases/tag/v${version}";
```

```suggestion
    changelog = "https://github.com/langchain-ai/langchain/releases/tag/langchain-mongodb==${version}";
```

Can you also check the other packages for similar things?